src/rdb.c

```diff
@@ -3220,6 +3226,7 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
                      * slot holds. */
                     if (slot_size) kvstoreHashtableExpand(db->keys, slot_id, slot_size);
                     if (expires_slot_size) kvstoreHashtableExpand(db->expires, slot_id, expires_slot_size);
+                    if (keys_with_volatile_items_slot_size) kvstoreHashtableExpand(db->keys_with_volatile_items, slot_id, keys_with_volatile_items_slot_size);
```

Comment:
It's a long line...

Not sure if my suggestion is too many lines instead. You can decide to ignore me.

```suggestion
                    if (keys_with_volatile_items_slot_size) {
                        kvstoreHashtableExpand(db->keys_with_volatile_items,
                                               slot_id,
                                               keys_with_volatile_items_slot_size);
                    }
```